### PR TITLE
VOWSoft.iBackupBot version 5.6.2

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21064)

### CODE REVIEW

File: manifests/v/VOWSoft/iBackupBot/5.6.2/VOWSoft.iBackupBot.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: VOWSoft.iBackupBot
+PackageVersion: 5.6.2
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
```
Comment (OfficialEsco) [inline manifests/v/VOWSoft/iBackupBot/5.6.2/VOWSoft.iBackupBot.installer.yaml:8]:
```suggestion
  InstallerType: nullsoft
  Scope: machine
```

File: manifests/v/VOWSoft/iBackupBot/5.6.2/VOWSoft.iBackupBot.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: VOWSoft.iBackupBot
+PackageVersion: 5.6.2
+PackageLocale: en-US
+Publisher: VOW Software Studio
+PackageName: iBackupBot
```
Comment (ItzLevvie) [inline manifests/v/VOWSoft/iBackupBot/5.6.2/VOWSoft.iBackupBot.locale.en-US.yaml:8]:
```suggestion
PackageName: iBackupBot for Windows
```

Comment (OfficialEsco) [inline manifests/v/VOWSoft/iBackupBot/5.6.2/VOWSoft.iBackupBot.locale.en-US.yaml:8]:
```suggestion
Publisher: VOW Software
PackageName: iBackupBot for Windows
```

File: manifests/v/VOWSoft/iBackupBot/5.6.2/VOWSoft.iBackupBot.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: VOWSoft.iBackupBot
+PackageVersion: 5.6.2
+PackageLocale: en-US
+Publisher: VOW Software Studio
```
Comment (ItzLevvie) [inline manifests/v/VOWSoft/iBackupBot/5.6.2/VOWSoft.iBackupBot.locale.en-US.yaml:7]:
```suggestion
Publisher: VOW Software
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/v/VOWSoft/iBackupBot/5.6.2/VOWSoft.iBackupBot.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [inline manifests/v/VOWSoft/iBackupBot/5.6.2/VOWSoft.iBackupBot.installer.yaml:8]:
```suggestion
  InstallerType: nullsoft
  Scope: machine
```
`Scope: machine` is missing here.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**